### ISSUE TITLE
Install issue on M2 Max

### ISSUE THREAD

**Issue by edit0r1593** [XingangPan/DragGAN#147]
I receive error during installation of pip dependencies:

ERROR: Invalid requirement: 'scipy=1.11.0'

It appears that the file created 'condaenv.20ea6o9b.requirements.txt' during the install requires ==

**Comment by ealton** [comment]
i got around it by moving scipy to pip section in environment.yml

**Comment by edit0r1593** [comment]
Yeah I did that when receiving the error at the environment creating stage. After moving scipy to pip that solved it. But then I got hit with the above error.

**Comment by glmcz** [comment]
On Mac M1 I just decreased the version of scipy to scipy=1.10.1 and was able to run demo with lions.
